makefiles/platform/Darwin-arm.mk

```diff
@@ -3,7 +3,13 @@ _THEOS_PLATFORM_LOADED := 1
 THEOS_PLATFORM_NAME := iphone
 
 _THEOS_PLATFORM_DEFAULT_TARGET := iphone
-_THEOS_PLATFORM_DPKG_DEB := dpkg-deb
+_THEOS_PLATFORM_DPKG_DEB := $(shell \
+	PATH=$$PATH:$(THEOS_BIN_PATH); \
+	DM=$$(which dm.pl); \
+	[ -z "$${DM}" -a -f "$(THEOS_BIN_PATH)/dm.pl" ] && DM="$(THEOS_BIN_PATH)/dm.pl"; \
+	[ ! -z "$${DM}" ] && ($(PERL) -MIO::Compress::Lzma -e '1' 2>/dev/null) \
```

Comment:
Does `$(PERL)` exist without #302?

Reply:
I replaced this with just `perl`, I think it’s just a remnant of the workarounds. Should work without it now that we have the hashbang workaround in bash.